types/src/v2/error.rs

```diff
@@ -84,6 +98,8 @@ pub const PARSE_ERROR_CODE: i32 = -32700;
 pub const OVERSIZED_REQUEST_CODE: i32 = -32701;
 /// Oversized response error code.
 pub const OVERSIZED_RESPONSE_CODE: i32 = -32702;
+/// Oversized response error code.
+pub const INVALID_SUBSCRIPTION_CODE: i32 = -32703;
```

Comment:
BTW, technically these error codes should be used for application errors.

To be really nitpicky I guess we should use `-32000 to -32099 Server defined errors here`

Reply:
Given this is a new code, do you think we should do the right thing for new error codes and stay within the right range?

Reply:
yeah, I used `32002` for this